server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -350,13 +337,57 @@ fn generate_response_requisition_lines(
                 expiring_units,
                 days_out_of_stock,
                 option_id,
-                // Default
-                supply_quantity: 0.0,
-                approved_quantity: 0.0,
-                approval_comment: None,
+                price_per_unit,
             },
-        )
-        .collect();
+        item_row: ItemRow { id: item_id, .. },
+        requisition_row: _,
+    } in request_lines
+    {
+        let price_per_unit = {
```

Comment:
Why not call `get_pricing_for_item` here? I was thinking that's where we'd put the variation for different setups and possibly where we'd add a plugin interface. This approach saves a db lookup but looking at the code now, I'm wondering why we don't just always populate the price, the  pref just controls if you see it?